tests/caasp/first_boot.pm

```diff
@@ -20,7 +20,12 @@ use caasp 'process_reboot';
 
 sub run {
     # On VMX images bootloader_uefi eats grub2 needle
-    assert_screen 'grub2' unless is_caasp('VMX');
+    if (check_var('FLAVOR', 'CaaSP-DVD-Incidents')) {
```

Comment:
Also here: Not very clear why QAM tests should behave different.

Reply:
I think better will be the simple increased timeout in both cases without the check of FLAVOUR,
btw 1200sec timeout for  GRUB needle? looks more like a problem in product